### ISSUE TITLE
istio-pilot fails to deploy

### ISSUE THREAD

**Issue by VariableDeclared** [canonical/istio-operators#42]
Hi 

When deploying the istio bundle, the following error is thrown:
```
creating or updating custom resource definitions: ensuring custom resource definit ion "meshpolicies.authentication.istio.io" with version "v1beta1": the server could not find the requested resource
```

Kubernetes version: 1.22.4-3+adc4115d990346 (Microk8s)
Deployed using repo instructions

Thanks

Peter

**Comment by DomFleischmann** [comment]
The version available on charmstore is `istio 1.5` which does not support kuberenetes 1.22 due to the API removals. The new istio oprator which is currently merged in Github uses `istio 1.11` and supports 1.22. It is not yet entirely published on charmhub but you can build it locally and try it out

**Comment by VariableDeclared** [comment]
thanks
